### PR TITLE
CLI + FIXES: Steward CLI

### PR DESCRIPTION
**Pull request by coachchucksol**
This PR adds the Steward CLI that can create and crank through the entire Steward state machine.

It also introduces some fixes to the Steward program.

### CODE REVIEW

File: utils/steward-cli/errors.txt [hunk 1]
```diff
@@ -0,0 +1,1246 @@
+
+Error: 3007
```
Comment (ebatsell) [inline utils/steward-cli/errors.txt:2]: remove this file

File: utils/steward-cli/temp.rs [hunk 2]
```diff
@@ -0,0 +1,231 @@
+use std::sync::Arc;
```
[the remark this answers is not in the document]
Comment (coachchucksol) [reply utils/steward-cli/temp.rs:1]: Done

File: utils/steward-cli/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    let _ = match args.commands {
+        // ---- Views ----
+        Commands::ViewConfig(args) => command_view_config(args, &client, program_id).await,
+        Commands::ViewState(args) => command_view_state(args, &client, program_id).await,
+        Commands::ViewNextIndexToRemove(args) => {
+            command_view_next_index_to_remove(args, &client, program_id).await
+        }
+        Commands::ViewStatePerValidator(args) => {
+            command_view_state_per_validator(args, &client, program_id).await
+        }
```
[the remark this answers is not in the document]
Comment (coachchucksol) [reply utils/steward-cli/src/main.rs:50]: Done

File: utils/steward-cli/src/utils/accounts.rs [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+pub async fn get_steward_config_account(
+    client: &RpcClient,
+    steward_config: &Pubkey,
+) -> Result<Config> {
+    let config_raw_account = client.get_account(steward_config).await?;
+
+    Ok(
+        Config::try_deserialize(&mut config_raw_account.data.as_slice())
+            .expect("Cannot deserialize config account"),
```
[the remark this answers is not in the document]
Comment (coachchucksol) [reply utils/steward-cli/src/utils/accounts.rs:64]: Done

File: utils/steward-cli/src/commands/command_args.rs [hunk 5]
```diff
@@ -0,0 +1,371 @@
...
+    #[arg(long, env)]
+    pub minimum_stake_lamports: Option<u64>,
+
+    /// Minimum number of consecutive epochs a validator has to vote before it can be considered for the pool
+    #[arg(long, env)]
+    pub minimum_voting_epochs: Option<u64>,
+}
+
+impl ConfigParameters {
+    pub fn to_update_parameters_args(&self) -> UpdateParametersArgs {
```
[the remark this answers is not in the document]
Comment (coachchucksol) [reply utils/steward-cli/src/commands/command_args.rs:107]: Done - thanks for the suggestion. Haven't played with `From` much

File: utils/steward-cli/errors.txt (hunk 1 above)
Comment (coachchucksol) [reply utils/steward-cli/errors.txt:2]: Done

**Pull request merged**